app/assets/stylesheets/hyrax/_work-show.scss

```diff
@@ -56,3 +56,24 @@ ul.tabular {
   margin-bottom: 18px;
   margin-top: -8px;
 }
+
+.panel-workflow {
```

Comment:
why are we defining this class in two places?

Reply:
I was following the perceived convention that part of this was including position (hence in the _position file). But I'm happy to do it only in one place.

Reply:
I think it's all a mismash, but please let's not define something twice.

Reply:
For sure. Fixed.